openlibrary/templates/type/work/editions_datatable.html

```diff
@@ -30,7 +30,7 @@
     </tbody>
 </table>
 <p class="small sansserif edition-adder">
-  <a href="/books/add?work=$work.key" title="$_('Add another edition of %(work)s', work=work.title)">$_("Add another edition")</a>?
+  <a href="/books/add?work=$work.key" title="$_('Add another edition of %(work)s', work=work.title)">$_("Add another edition?")</a>
```

Comment:
Since we've changed the msgid here, need to update the other languages to also have `?` in the msgid, otherwise there translations will stop working.

Reply:
This actually fixes this string for French readers.  The Polish `msgid` also has a question mark, but the translation is fuzzy and not displayed to readers...

The Croatian `msgid` needs a question mark, which could change the translation.  This string is untranslated in other languages.